packages/core/strapi/src/services/entity-service/index.ts

```diff
@@ -337,7 +373,11 @@ const createDefaultImplementation = ({
 
     const model = strapi.getModel(uid);
 
-    const entityToClone = await db.query(uid).findOne({ where: { id: cloneId } });
+    const populateValue = getMediaPopulate(uid);
+
+    const entityToClone = await db
+      .query(uid)
+      .findOne({ where: { id: cloneId }, populate: populateValue });
```

Comment:
We might want to explain why this needs to be done too:
```suggestion
	/**
	 * Media attributes are not automatically cloned in the db layer,
	 * this populate ensures media is cloned in case user does not provide
	 * any override value
	 */
    const populate = getMediaPopulate(uid);

    const entityToClone = await db
      .query(uid)
      .findOne({ where: { id: cloneId }, populate });
```